src/setup/devnet/index.js

```diff
@@ -686,8 +687,8 @@ export class Devnet {
           }
           // copy the Ganache files to the first node
 
-          const ganacheURL = new URL(this.config.ethURL)
-          const ganacheUser = this.config.ethHostUser
+          const anvilURL= new URL(this.config.ethURL)
```

Comment:
I see ganache still imported and heavily used in this and other files. Let's make sure to remove it everywhere.